src/sentry/pipeline/__init__.py

```diff
@@ -226,6 +226,8 @@ def current_step(self):
 
     def error(self, message):
         context = {"error": message}
+        logger = self.get_logger()
+        logger.warn("pipeline error %s" % message)
```

Comment:
My sense is that once we get this error, we'll want to know whom this happened to and if it was from Slack or not. Any way we can capture that information? Can we log self.organization.id and the provider if those are around?

Reply:
@manuzope I was hoping that as a Sentry error, we'd get the full context of the pipeline object but that doesn't appear to be so. Also changed the level to `error` since I realize I was wrong about seeing warnings in Sentry for the Sentry project.